### DISCUSSION TITLE
Music

### DISCUSSION THREAD

**Discussion by legal-robotics-user** [blueedgetechno/win11React#335] [General]
whats the "brand in free fire" song in spotify? i looked it up and google yielded no results. i'm a scratch coder and would like to use it in a scratch project for DeV music (https://scratch.mit.edu/studios/31210598). My username is NOBL-e, so if you have an answer go to https://scratch.mit.edu/users/NOBL-e/ and post your answer.

**Comment by luni-moon** [comment]
> whats the "brand in free fire" song in spotify? i looked it up and google yielded no results. i'm a scratch coder and would like to use it in a scratch project for DeV music (https://scratch.mit.edu/studios/31210598). My username is NOBL-e, so if you have an answer go to https://scratch.mit.edu/users/NOBL-e/ and post your answer.

It may be copyrighted, so even if you find the song, you may need to pay the artist a sum of money for a license to use it in a project, no matter if used for monetary gain or just for fun.

**Reply by legal-robotics-user** [reply]
i could perform it myself- i was in choir like a year ago. ive recorded myself and i sound good.

**Reply by legal-robotics-user** [reply]
:]

**Reply by luni-moon** [reply]
You'd still have to pay - or provide some other form of compensation - since it was originally their song. Your school, or place where you were in a choir, most likely paid for the music online. As a music artist, who writes remixes, which is similar to performing the song yourself, I still have to get permission from the original artist to publish my remixes.
